campaign-launcher/client/src/components/Footer/index.tsx

```diff
@@ -35,6 +35,10 @@ const StyledLink = styled(Link)(({ theme }) => ({
   },
 }));
 
+const handleClickOnSocialButton = (url: string) => {
```

Comment:
It could be better to pass `url` as a prop to `SocialMediaIconButton` and have this `handleClickOnSocialButton` as internal logic